hazelcast-client/src/test/java/com/hazelcast/client/map/impl/nearcache/ClientMapNearCacheTest.java

```diff
@@ -767,4 +717,18 @@ protected NearCacheConfig newNoInvalidationNearCacheConfig() {
     protected ClientConfig newClientConfig() {
         return new ClientConfig();
     }
+
+    protected void assertNearCacheInvalidation_whenMaxSizeExceeded(NearCacheConfig config) {
+        final IMap<Integer, Integer> map = getNearCachedMapFromClient(config);
+        populateNearCache(map, MAX_CACHE_SIZE);
```

Comment:
Previously this number was MAX_CACHE_SIZE * 2. If that serves a purpose we may have caused a change in what is being tested. What is the affect of this change?